tests/FSharp.Compiler.ComponentTests/Conformance/Constraints/Unmanaged.fs

```diff
@@ -485,6 +501,71 @@ printf "%s" (CsharpStruct<int>.Hi<MultiCaseUnion>())
     IL_0000:  ret
   } """]
 
+    [<Fact>]
+    let ``FSharp does not generate modreq for VBNET to consume in v8`` () = 
```

Comment:
Get rid of this test function alltogether.
We do not need exact duplicates with one having CSharp in the name and other VBNET. Just keep the Csharp one, fully remove this function.